pystiche_papers/li_wand_2016/_loss.py

```diff
@@ -105,6 +124,36 @@ def style_loss(
     target_transforms: Optional[Iterable[transforms.Transform]] = None,
     score_weight: Optional[float] = None,
 ) -> ops.MultiLayerEncodingOperator:
+    r"""Style_loss from :cite:`LW2016`.
+
+    Args:
+        impl_params: If ``True``, uses the parameters used in the reference
+            implementation of the original authors rather than what is described in
+            the paper. For details see below.
+        multi_layer_encoder: Pretrained :class:`~pystiche.enc.MultiLayerEncoder`. If
+            omitted,  the default
+            :func:`~pystiche_papers.li_wand_2016.multi_layer_encoder` is used. Defaults
+            to ``None``.
+        layers: Layers from which the encodings of the ``multi_layer_encoder`` should be
+            taken. If omitted, the defaults is used. Defaults to
+            ``("relu3_1", "relu4_1")``.
+        layer_weights: Layer weights of the operator. Defaults to ``"sum"``.
+        patch_size: Size of the patch.
```

Comment:
Please explain what happens if this is given as `int` rather than as tuple of two `int`s. See for example https://pytorch.org/docs/stable/generated/torch.nn.Conv2d.html?highlight=conv2d#torch.nn.Conv2d